src/main/java/mate/academy/spring/service/impl/UserServiceImpl.java

```diff
@@ -1,27 +1,24 @@
 package mate.academy.spring.service.impl;
 
 import java.util.Optional;
+import mate.academy.spring.config.SecurityConfig;
 import mate.academy.spring.dao.UserDao;
 import mate.academy.spring.model.User;
 import mate.academy.spring.service.UserService;
-import mate.academy.spring.util.HashUtil;
 import org.springframework.stereotype.Service;
 
 @Service
 public class UserServiceImpl implements UserService {
-    private static final int SALT_LENGTH = 10;
     private final UserDao userDao;
+    private final SecurityConfig securityConfig;
 
-    public UserServiceImpl(UserDao userDao) {
+    public UserServiceImpl(UserDao userDao, SecurityConfig securityConfig) {
```

Comment:
Why do you need this?

Reply:
didnt think i could use PasswordEncoder directly